mezzo/src/data/tracks.json

```diff
@@ -1,4 +1,100 @@
 [
+    {
```

Comment:
Can we .gitignore anything in data folder please :)

Reply:
yes!